packages/ocephes/ocephes.0.0.1/opam

```diff
@@ -0,0 +1,11 @@
+opam-version: "1.2"
+maintainer: "Leonid Rozenberg <leonidr@gmail.com>"
+authors: "Leonid Rozenberg <leonidr@gmail.com>"
+homepage: "https://github.com/rleonid/Ocephes"
+bug-reports: "https://github.com/rleonid/Ocephes/issues"
+license: "Apache 2.0"
+dev-repo: "https://github.com/rleonid/Ocephes.git"
+build: [make]
+install: [make "install"]
+remove: ["make uninstall"]
+depends: ["ocamlfind" "ctypes"]
```

Comment:
It looks like you also depend on `oasis {build}` right now. Either pre-generate and distribute the portable oasis build products or add `oasis {build}` as a dependency. `ocamlfind` could also be made a `{build}` dependency.